### ISSUE TITLE
unable to init unauthenticated git submodules since March 15, 2022

### ISSUE THREAD

**Issue by francesco-ballarin** [ngsxfem/ngsxfem#11]
Git repo urls in https://github.com/ngsxfem/ngsxfem/blob/release/.gitmodules should be changed to use `https://` due to https://github.blog/2021-09-01-improving-git-protocol-security-github/

Since March 15, current main branch gives errors like the following:
```
+ cd /tmp/ngsxfem-src/
+ git checkout master
Switched to a new branch 'master'
branch 'master' set up to track 'origin/master'.
+ git submodule update --init
Submodule 'external_dependencies/jupyter' (https://github.com/ngsxfem/ngsxfem-jupyter) registered for path 'external_dependencies/jupyter'
Submodule 'external_dependencies/ngsolve' (git://github.com/NGSolve/ngsolve.git) registered for path 'external_dependencies/ngsolve'
Cloning into '/tmp/ngsxfem-src/external_dependencies/jupyter'...
Cloning into '/tmp/ngsxfem-src/external_dependencies/ngsolve'...
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
fatal: clone of 'git://github.com/NGSolve/ngsolve.git' into submodule path '/tmp/ngsxfem-src/external_dependencies/ngsolve' failed
Failed to clone 'external_dependencies/ngsolve'. Retry scheduled
Cloning into '/tmp/ngsxfem-src/external_dependencies/ngsolve'...
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
fatal: clone of 'git://github.com/NGSolve/ngsolve.git' into submodule path '/tmp/ngsxfem-src/external_dependencies/ngsolve' failed
Failed to clone 'external_dependencies/ngsolve' a second time, aborting
```

**Comment by schruste** [comment]
Dear Francesco,
Thanks for the hint. It's fixed with commit 9705ed7 (in the master - a release is coming soon anyway).
Best, Christoph

**Comment by francesco-ballarin** [comment]
Thanks. Fixed by https://github.com/ngsxfem/ngsxfem/commit/9705ed728d8d14114fdef560a9dfd9dcc63dcffc.
